cpp/include/cudf/stream_compaction.hpp

```diff
@@ -296,10 +291,10 @@ cudf::size_type distinct_count(column_view const& input,
  * @param[in] nulls_equal flag to denote if null elements should be considered equal.
  *            nulls are not equal if null_equality::UNEQUAL.
  *
- * @return number of distinct consecutive groups in the table
+ * @return number of consecutive groups of equivalent rows in the column
  */
-cudf::size_type distinct_count(table_view const& input,
-                               null_equality nulls_equal = null_equality::EQUAL);
+cudf::size_type unique_count(table_view const& input,
+                             null_equality nulls_equal = null_equality::EQUAL);
 
 /**
  * @brief Count the unique elements in the column_view.
```

Comment:
Avoid "unique" in `distinct_count` definition:
```suggestion
 * @brief Count the distinct elements in the column_view.
```
The docstring body also needs fixed:
```diff
- * Given an input column_view, number of unique elements in this column_view is returned.
+ * Given an input column_view, number of distinct elements in this column_view is returned.
```

```diff
- * `nan_handling` is nan_policy::NAN_IS_VALID, only `null` is ignored, `NaN` is considered in unique
+ * `nan_handling` is nan_policy::NAN_IS_VALID, only `null` is ignored, `NaN` is considered in distinct
```

```diff
- * @param[in] input The column_view whose unique elements will be counted
+ * @param[in] input The column_view whose distinct elements will be counted
```